cli/azd/extensions/microsoft.azd.extensions/internal/cmd/init.go

```diff
@@ -598,7 +598,7 @@ func createExtensionDirectory(
 		return fmt.Errorf("failed to copy and process templates: %w", err)
 	}
 
-	if extensionMetadata.Language == "dotnet" || extensionMetadata.Language == "javascript" {
```

Comment:
Lets keep the proto files for javascript and also add python so that users have a better path for regenerating new client libraries as the proto files evolve.

Reply:
We changed it to: `if extensionMetadata.Language != "go" {`.